chore: drop support for PHP 7.4 (#558)
firebase/php-jwt: README.md

composer require firebase/php-jwt
```

> Optionally, install the `paragonie/sodium_compat` package from composer if you
> do not have libsodium installed:

Reviewer: nit
```suggestion
Optionally, install the `paragonie/sodium_compat` package from composer if your
php env does not have libsodium installed:
```
